### DISCUSSION TITLE
audits.id as UUID

### DISCUSSION THREAD

**Discussion by leandroneves-net** [owen-it/laravel-auditing#920] [Q&A] [answered]
Hello, 

I would like to use audits.id as UUID. Is it possible? Is it configurable or I need write a patch?

Thanks in advance,

Leandro

**Comment by parallels999** [comment, marked as answer]
Update your migration file
You have to make your own custom model extending `OwenIt\Auditing\Models\Audit`(_with UUID changes_), and set it in `audit.implementation` on config
https://github.com/owen-it/laravel-auditing/blob/22682dd5c800aab90ef261530098625e26a66709/config/audit.php#L16
https://laravel-auditing.com/guide/audit-migration.html#uuid-over-auto-incrementing-ids
